### ISSUE TITLE
Deploy using Google's App Engine (Platform as a Service)

### ISSUE THREAD

**Issue by slavakurilyak** [produvia/kryptos#61]
# User Story

As a developer, I want to deploy Kryptos to [Google's App Engine](https://cloud.google.com/appengine/), which offers Docker Image support, so that I run a highly scalable application on a fully managed serverless platform powered by Google Cloud.

# Inspiration

Google's App Engine was released in 2008 with the goal of allowing "developers to focus on doing what they do best, writing code." ([Google, 2018](https://cloud.google.com/appengine/docs/flexible/))

# Note

> If you deploy applications on Google App Engine (GAE) you will start to notice that you are spending money for two instances instead of just one when your application *is* running. ([Medium, 2017](https://medium.com/google-cloud/scaling-google-app-engine-to-no-instances-or-maybe-just-1-37be4e8d4230))

In other words, Google App Engine sets up **two** VMs at the beginning and offers two environments (Flexible and Standard) for finer control. 

"If you scaled down the number of instances in your Flexible Environment from two to one", you will "cut your GAE bill in half." ([Medium, 2017](https://medium.com/google-cloud/scaling-google-app-engine-to-no-instances-or-maybe-just-1-37be4e8d4230))
